next.clayui.com/content/docs/css/components/tabs.md

```diff
@@ -0,0 +1,634 @@
+                            <use href="/images/icons/icons.svg#check"></use>
+                        </svg>
+                    </span>
+                </button>
+            </li>
+            <li>
+                <button aria-controls="buttonSupportedClients" class="dropdown-item" data-target="#buttonSupportedClients" data-toggle="tab" id="supportedClientsTab" role="tab">
+                    Supported Clients
+                    <span aria-hidden="true" class="dropdown-item-indicator-end">
+                        <svg class="lexicon-icon lexicon-icon-check" focusable="false" role="presentation">
+                            <use href="/images/icons/icons.svg#check"></use>
+                        </svg>
+                    </span>
+                </button>
+            </li>
+        </ul>
+    </li>
+</ul>
+<div class="tab-content">
+    <div aria-labelledby="buttonFieldsTab" class="active fade show tab-pane" id="buttonFields" role="tabpanel">
+        Single origin, extra id beans, eu to go, skinny americano ut aftertaste sugar. At americano, viennese variety iced grounds, grinder froth and pumpkin spice aromatic. Cultivar aged lungo, grounds café au lait, skinny, blue mountain, in variety sugar shop roast. Wings, blue mountain affogato organic cappuccino java plunger pot. Single shot variety pumpkin spice seasonal skinny barista carajillo robust cream.
+    </div>
+    <div aria-labelledby="buttonSettingsTab" class="fade tab-pane" id="buttonSettings" role="tabpanel">
+        Iced, crema, coffee id kopi-luwak coffee variety. Skinny extraction, id trifecta qui trifecta grinder. Barista robusta arabica breve ut skinny milk beans macchiato carajillo espresso. Cultivar single shot brewed, coffee steamed to go wings to go cortado. Grinder, siphon coffee acerbic espresso cinnamon crema breve.
+    </div>
+    <div aria-labelledby="buttonDocumentsTab" class="fade tab-pane" id="buttonDocuments" role="tabpanel">
+        Cultivar single shot brewed, coffee steamed to go wings to go cortado. Grinder, siphon coffee acerbic espresso cinnamon crema breve.
+    </div>
+    <div aria-labelledby="buttonSetupTab" class="fade tab-pane" id="buttonSetup" role="tabpanel">
+        Skinny extraction, id trifecta qui trifecta grinder. Barista robusta arabica breve ut skinny milk beans macchiato carajillo espresso. Cultivar single shot brewed, coffee steamed to go wings to go cortado. Grinder, siphon coffee acerbic espresso cinnamon crema breve.
+    </div>
+    <div aria-labelledby="buttonSupportedClientsTab" class="fade tab-pane" id="buttonSupportedClients" role="tabpanel">
+        At americano, viennese variety iced grounds, grinder froth and pumpkin spice aromatic. Cultivar aged lungo, grounds café au lait, skinny, blue mountain, in variety sugar shop roast. Wings, blue mountain affogato organic cappuccino java plunger pot. Single shot variety pumpkin spice seasonal skinny barista carajillo robust cream.
+    </div>
+</div>
+```
+
+## Justified
+
```

Comment:
We should probably add a description here and what class name is needed

Reply:
Yep. Adding docs for this!